src/api/routes/series.js

```diff
@@ -227,4 +230,93 @@ router.get('/:bbid/relationships',
 		return res.status(200).send(seriesRelationshipList);
 	});
 
+/**
+ *	@swagger
+ * '/series':
+ *   get:
+ *     tags:
+ *       - Browse Requests
+ *     summary: Gets a list of Series related to another Entity
+ *     description: BBID of an Author or an Edition or an EditionGroup or a Publisher or a Work is passed as query parameter and it's Series are fetched
+ *     operationId: getRelatedSeriesByBbid
+ *     parameters:
+ *       - name: author
+ *         in: query
+ *         description: BBID of the corresponding Author
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: work
+ *         in: query
+ *         description: BBID of the corresponding Work
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: edition
+ *         in: query
+ *         description: BBID of the corresponding Edition
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: edition-group
+ *         in: query
+ *         description: BBID of the corresponding Edition Group
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: publisher
+ *         in: query
+ *         description: BBID of the corresponding Publisher
+ *         required: false
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *       - name: seriesOrderingType
+ *         in: query
+ *         description: filter by Series Ordering Type
+ *         required: false
+ *         schema:
+ *           type: string
+ *           enum: [Automatic, Manual]
+ *     responses:
+ *       200:
+ *         description: List of Series related to another Entity
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/BrowsedSeries'
+ *       404:
+ *         description: author/work/edition/edition-group/publisher (entity entity) not found
+ *       400:
+ *         description: Invalid BBID passed in the query params OR Multiple browsed entities passed in parameters
+ */
+
+router.get('/',
+	formatQueryParameters(),
+	validateBrowseRequestQueryParameters(['edition', 'author', 'edition-group', 'work', 'publisher']),
+	makeEntityLoader(null, utils.relationshipsRelations, 'Entity not found', true),
+	loadEntityRelationshipsForBrowse(),
+	async (req, res) => {
+		function relationshipsFilterMethod(relatedEntity) {
+			if (req.query.seriesorderingtype) {
+				return toLower(relatedEntity.seriesOrderingType) === toLower(req.query.seriesorderingtype);
```

Comment:
There is a mismatch of the capitalization here. Does the query parameter name get automatically transformed from `seriesOrderingType` (as defined in the docs on line 278) to `seriesorderingtype` as seen here on the query object?

The filtering seems to me to be working fine, so we should probably correct the docs?

Reply:
> Does the query parameter name get automatically transformed from seriesOrderingType (as defined in the docs on line 278) to seriesorderingtype as seen here on the query object ?

Yes, the middleware [formatQueryParameters](https://github.com/bookbrainz/bookbrainz-site/blob/66a1b8706579eb1bfa8b598e0071bf239c408fa9/src/api/helpers/middleware.js#L79-L84) transforms all the query parameters to lowerCase.